src/components/AccountDrawer/AuthenticatedHeader.tsx

```diff
@@ -282,6 +283,8 @@ export default function AuthenticatedHeader({ account, openSettings }: { account
               {formatNumber({
                 input: totalBalance,
                 type: NumberType.PortfolioBalance,
+                locale: formatterLocale,
+                localCurrency: formatterLocalCurrency,
```

Comment:
feel like this would be nicer with a single hook:

```tsx
const { formatNumber } = useFormatter()

formatNumber({ input, type })
```

Reply:
or `useFormatNumber()` if we don't need other types of formatters / they have different performance costs

Reply:
that does sound better will impl that instead